sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClientIntegrationTest.java

```diff
@@ -927,12 +945,22 @@ void updateRuleResponse(HttpClient httpClient) {
     }
 
     private ServiceBusAdministrationAsyncClient createClient(HttpClient httpClient) {
+        return createClient(httpClient, false);
+    }
+
+    private ServiceBusAdministrationAsyncClient createClient(HttpClient httpClient, boolean isUnauthorizedClient) {
         final String connectionString = interceptorManager.isPlaybackMode()
             ? "Endpoint=sb://foo.servicebus.windows.net;SharedAccessKeyName=dummyKey;SharedAccessKey=dummyAccessKey"
             : TestUtils.getConnectionString();
+
```

Comment:
This is only one use-case. Rather than adding an entirely new overload, can you manually create the clients.
https://github.com/Azure/azure-sdk-for-java/pull/18518/files#diff-df3a009ad8b45047bb87f09a45e5730bcb9cfaaae1b86cf43650bdf98fb73061R810